TripPlanner - Project/TripPlannerFrontend/src/components/Buttons/ButtonPrimary.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components'
+
+const ButtonPrimary = styled.button`
+  padding: 10px 18px 10px 18px;
+  width: 105px;
+  height: 34px;
+  background-color: #F53314;
+  border: 0;
+  border-radius: 6px;
+  color: #fff;
+  cursor: pointer;
+  left: calc(50% - 105px/2);
+  top: calc(50% - 34px/2 + 54px);
```

Comment:
It looks like you've probably copied every style from Figma. Copying every style from Figma doesn't work. Why?

Figma is just a showcase for our design system. Components have to be composable and reusable. If you specify, `width`, `left` and `top`, they will not look properly when using texts of different lengths, etc.

Please make sure that components are reusable, not tied up to specific use case.